packages/gravity-ui-nunjucks/src/_patterns/00-particles/01-typography/00-fonts.njk

```diff
@@ -1,16 +1,18 @@
 {% for font in fonts %}
-  <h4 class="sg-pattern-sub-heading">{{font.name}}</h4>
+  <h4 class="sg-pattern-sub-heading">{{font.cssFontFamily}}</h4>
   <ul class="sg-fonts">
     {% for weight in font.weights %}
       <li class="sg-fonts__font">
         <strong class="sg-fonts__weight">{{weight.name}} weight</strong>
-        <code class="sg-fonts__css">font-family: {{font.cssFontFamily}}; font-weight: {{weight.cssFontWeight}}; font-style: normal;</code>
+        <span class="sg-fonts__css"><strong>CSS</strong>: <code>font-family: {{font.cssFontFamily}}; font-weight: {{weight.cssFontWeight}}; font-style: normal;</code></span>
+        <span class="sg-fonts__css"><strong>SCSS</strong>: <code>@include gravy-font({{font.name}}, {{weight.gravityWeight}});</code></span>
```

Comment:
I thought `gravy` was out? Is this something different or can it be removed?

Reply:
Ah! Nope, I just forgot to update that sample code. You're quite right, the `gravy-font()` mixin no longer exists.

I'll push a fix for that soon